### PR TITLE
alex - Admin Personal Schedule Placeholder Page and Navbar

### PR DESCRIPTION
**Pull request by AlexLaiUCSB**
# Navbar
image
# Placeholder Page w/ Route
image

# Placeholder Page in Storybook
image

### CODE REVIEW

**Review by andrewpengucsb — APPROVED** [review]
LGTM!

**Review by btk5h — CHANGES_REQUESTED** [review]
Overall good first PR, just some unused code that should be cleaned up.

File: frontend/src/tests/components/Nav/AppNavbar.test.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { currentUserFixtures } from "fixtures/currentUserFixtures";
 
 import AppNavbar from "main/components/Nav/AppNavbar";
 import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import { Dropdown } from "bootstrap";
```
[the remark this answers is not in the document]
Comment (AlexLaiUCSB) [reply frontend/src/tests/components/Nav/AppNavbar.test.js]: Yes, removed that import in new commit

**Review by btk5h — APPROVED** [review]

**Review by andrewpengucsb — DISMISSED** [review]
Approval for after the changes, LGTM!

**Review by andrewpengucsb — APPROVED** [review]
Good stuff, LGTM

**Review by andrewpengucsb — APPROVED** [review]
Placeholder looks good! LGTM

**Review by ScottUCSB — APPROVED** [review]
Looks like a solid placeholder page with a new entry on the Navbar

**Review by andrewhlu — APPROVED** [review]
LGTM! This is a solid 10 point issue!

**Pull request merged**